app-modules/prospect/src/Filament/Resources/ProspectResource.php

```diff
@@ -114,6 +114,7 @@ public static function getGlobalSearchResultDetails(Model $record): array
             'Other ID' => $record->otherid,
             'Email Address' => collect([$record->email, $record->email_id])->filter()->implode(', '),
             'Phone' => collect([$record->mobile, $record->phone])->filter()->implode(', '),
+            'Preferred Name' => $record->preferred,
```

Comment:
You also need to add `preferred` to `getGloballySearchableAttributes`.

Also, @danharrin should we include it in the `scoreGlobalSearchResults` in `modifyGlobalSearchQuery` in any way as well?

Reply:
Fixed

Reply:
No I don't think we need to score by this attribute for now